### PR TITLE
Initial attempt at CI doc build

### PR DESCRIPTION
**Pull request by brianmajor**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      run: test -f GMSImp.pdf
+        
+    - uses: actions/upload-artifact@v1
+      with:
+        name: GMS.pdf Preview
+        path: GMS.pdf
+        
+    - uses: actions/upload-artifact@v1
+      with:
+        name: DataLinkImp.pdf Preview
```
Comment (molinaro-m) [inline .github/workflows/build.yml]: shouldm't it be GMSImp.pdf also here?

**Review by molinaro-m — DISMISSED** [review]
As far as I recall CI from other repos, this looks fine.
I suppose there's no simpler way to test it but trying to merge it.

I left a small comment on a non-critical change in the build file.

**Review by molinaro-m — APPROVED** [review]
Fine, thanks for the update.

**Comment by brianmajor** [comment]
Thanks @molinaro-m , it worked!  I've made a reference to the artifact at the top of the GMS RFC page.

**Pull request merged**